index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
- In your README file provide information about you and your project. You could use [this template](https://github.com/microverseinc/readme-template).
Just fill this template with your information, information about your project, how to install it locally, provide a license... you could fill it following the steps provided there.